cartography/intel/crxcavator/__init__.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import os
+
+from cartography.intel.crxcavator.crxcavator import sync_extensions
+from cartography.util import run_cleanup_job
+
+logger = logging.getLogger(__name__)
+
+# API key for CRXcavator generaated from crxcavator.io portal
+CRXcavator_API_KEY = os.environ.get('CREDENTIALS_CRXCAVATOR_API_KEY')
```

Comment:
Use all-caps for the variable name here since it's a constant.

Reply:
fixed